test/h3core.spec.js

```diff
@@ -29,8 +29,8 @@ function toLowPrecision(maybeNumber) {
     throw new Error(`Unhandled type: ${maybeNumber}`);
 }
 
-function almostEqual(a, b, epsilon = Number.EPSILON * 10) {
-    return Math.abs(a - b) < epsilon;
+function almostEqual(a, b, factor = 1e-6) {
+    return Math.abs(a - b) < a * factor;
```

Comment:
Using a factor instead of an epsilon here avoids FPE issues in Node 6, and seems conceptually equivalent.